src/it/resources/logback.xml

```diff
@@ -2,11 +2,11 @@
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} %-5level %logger{36} - %msg%n</pattern>
```

Comment:
why?

Reply:
I think that in an actor model, it doesn't bring much value to know the thread we are running and it makes the logs smaller